src/components/HeaderMenu.js

```diff
@@ -28,8 +28,8 @@ function Header(token, setToken) {
         axios
             .post(
                 `https://teammate-app.herokuapp.com/auth/token/logout/`,
-                {},
-                {headers: { Authorization:`Token$ {token}`}}
+                {}, {withCredentials: true},
```

Comment:
What does line 31 do here?

Reply:
works like the token authorization, I was just trying something.

Reply:
Cool! I haven't seen that before. Do we need to change anything about login if we add this here?

Reply:
no we don't have to, but to be sure we can go back to use :  {headers: { Authorization:`Token$ {token}`}}